spec/support/models/label.rb

```diff
@@ -44,6 +44,6 @@ def after_validation_stub
 
   private
   def cleanup
-    self.name = self.name.downcase.capitalize
+    self.name = self.name.downcase.capitalize if self.name
```

Comment:
Test-only change that was causing saving a Label to crash when not setting a name.